clickhouse_driver/columns/datecolumn.py

```diff
@@ -7,7 +7,22 @@
 epoch_end = date(2149, 6, 6)
 
 epoch_start_date32 = date(1900, 1, 1)
-epoch_end_date32 = date(2283, 11, 11)
+# Why was it 17 year earlier that limit set in clickhouse?
```

Comment:
It seems that some old version has this limitation https://github.com/mymarilyn/clickhouse-driver/actions/runs/8805937959